dooit/utils/default_config.py

```diff
@@ -26,7 +26,13 @@ def get_clock() -> Text:
 
 
 def get_username():
-    return Text(f" {os.getlogin()} ", "r " + blue)
+    try:
+        username = os.getlogin()
+    except OSError:
+        uid = os.getuid()
+        import pwd
+        username = pwd.getpwuid(uid).pw_name
+    return Text(f" {username} ", "r " + blue)
```

Comment:
```suggestion
    if hasattr(os, getlogin):
        username = os.getlogin()
    else:
        uid = os.getuid()
        import pwd
        username = pwd.getpwuid(uid).pw_name
    return Text(f" {username} ", "r " + blue)
```

Reply:
With this change, we can not catch the Exception.

Reply:
Hmm... that might be true, haha I just changed that because Exceptions are slow but I don't think it'll matter much here :smile: 

~Maybe add one more general Exception statement to catch and print the error?~
Nah I don't think any more errors would show up! Alright then, I'll merge this
Sorry for the un-necessary delay :p